### ISSUE TITLE
 line 542, in <module>  msg.save(toJson, useFileName

### ISSUE THREAD

**Issue by richrumble** [mattgwwalker/msg-extractor#15]
The errors are as follows: (latest extractmsg.py btw)

$ python /usr/bin/ExtractMSG.py testemail-unicode-rtf.msg
Error with file 'testemail-unicode-rtf.msg': Traceback (most recent call last):
  File "/usr/bin/ExtractMSG.py", line 542, in <module>
    msg.save(toJson, useFileName)
  File "/usr/bin/ExtractMSG.py", line 413, in save
    attachmentNames.append(attachment.save())
  File "/usr/bin/ExtractMSG.py", line 193, in save
    f.write(self.data)
TypeError: argument 1 must be string or buffer, not None

Same for all the other msg's I've attached. Outlook version 2007 fully patched.
I get the same with 2010 msg's as well. The errors are also present when using --raw as well. 
[testemails.zip](https://github.com/mattgwwalker/msg-extractor/files/630966/testemails.zip)

**Comment by caje0004** [comment]
Same problem with Outlook version 2016.
Errors in lines: 542, 432 and 408

**Comment by realaxl4ibm** [comment]
This solved an encoding error for me, may help for you as well:

Include UTF-8 codecs, add to includes at top of file:

```
import codecs
```

Replace @ line 405:
```
            # f = open("message." + fext, "w")
            f = codecs.open("message." + fext, "w", "utf-8") # UTF-8 file 
```

**Comment by TheElementalOfCreation** [comment]
Should be fixed in version 0.15. Please test it to confirm.

**Comment by TheElementalOfCreation** [comment]
Should this issue not be fixed, please re-open the issue.
